src/styles/main.scss

```diff
@@ -0,0 +1,157 @@
+@import 'colors';
+@import 'measures';
+
+.container {
+  padding: $paddingV-card $paddingH-card;
+  display: grid;
+  grid-template-columns: repeat(4, $width-card);
+  gap: $row-gap $column-gap;
+  justify-content: center;
+  box-sizing: border-box;
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $width-card);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $width-card);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $width-card);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $width-card;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
+    margin: 0 16px 4px;
+    font-family: $roboto-medium;
+    font-weight: $roboto-weight-medium;
+    color: $color-font;
+
+    &:hover {
+      color: $color-title-hover;
+    }
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 16px 16px;
+    font-family: $roboto-medium;
+    font-weight: $roboto-weight-regular;
+    min-height: 14px;
+    max-width: 98px;
```

Comment:
The `max-width` property for `.card__subtitle` might not be necessary if the text is expected to wrap and the container's width is controlled by its parent. If `max-width` is required, ensure it's responsive and doesn't cause issues on smaller screens.